tjreports/site/views/reports/tmpl/default.php

```diff
@@ -187,7 +186,7 @@ class='icon-download'></span>" . JText::_('COM_TJREPORTS_CSV_EXPORT') . "</a>";
 					</div>
 
 					<div>
-					<div class="col-sm-8 col-xs-12 offset1">
+					<div class="col-sm-8 col-xs-12">
```

Comment:
what do you want to use? bootstrap 2 or 3? If bootstrap 2, use row-fluid and spans..